app/assets/stylesheets/application.scss

```diff
@@ -5,16 +5,23 @@
 @import "colors";
 @import 'sizes';
 @import 'mixins';
-@import 'new_styles/new_mixins';
 
 /* core */
+@import 'media-box';
+@import 'autocomplete';
```

Comment:
@salbertson SCSS linting is [turned off](https://github.com/diaspora/diaspora/blob/develop/.hound.yml#L8) but as you can see Hound commented on SCSS code after a forced push.